pegasus-spark-bulkloader/src/main/java/com/xiaomi/infra/pegasus/spark/bulkloader/BulkLoaderConfig.java

```diff
@@ -0,0 +1,131 @@
+package com.xiaomi.infra.pegasus.spark.bulkloader;
+
+import com.xiaomi.infra.pegasus.spark.CommonConfig;
+import com.xiaomi.infra.pegasus.spark.FDSConfig;
+import com.xiaomi.infra.pegasus.spark.HDFSConfig;
+import java.io.Serializable;
+
+public class BulkLoaderConfig extends CommonConfig {
```

Comment:
This is the only user-facing interface of bulkloader. I think comments need more to be here.
Users need additional info about the destination of the generated files, but they are not documented here.

```suggestion
// The bulk-load generated files are placed as following:
//
// <DataPathRoot>/<ClusterName>/<TableName>
// <DataPathRoot>/<ClusterName>/<TableName>/bulk_load_info => { JSON }
// <DataPathRoot>/<ClusterName>/<TableName>/<PartitionIndex>/bulk_load_metadata => { JSON }
// <DataPathRoot>/<ClusterName>/<TableName>/<PartitionIndex>/<FileIndex>.sst => RocksDB SST File
public class BulkLoaderConfig extends CommonConfig {
```

Please check the comments above and ensure I didn't make thing wrong.